internal/backend/azure/config.go

```diff
@@ -16,6 +16,7 @@ type Config struct {
 	AccountName string
 	AccountSAS  options.SecretString
 	AccountKey  options.SecretString
+	Domain      string
```

Comment:
In the azure documentation, the parameter is called (storage) endpoint suffix.
https://learn.microsoft.com/en-us/azure/storage/common/storage-powershell-independent-clouds#get-endpoint-using-get-azenvironment

What about renaming it to `EndpointSuffix` and adjusting the environment variable name accordingly?